handbook/engineering/README.md

```diff
@@ -543,13 +543,21 @@ You can read our guide to diagnosing issues in Fleet on the [debugging page](htt
 
 ## Bug process
 
+- [Creating bugs](#creating-bugs)
 - [Bug states](#bug-states)
 - [Finding bugs](#finding-bugs)
 - [Outages](#outages)
 - [All bugs](#all-bugs)
 
 All bugs in Fleet are tracked by QA on the [bugs board](https://app.zenhub.com/workspaces/-bugs-647f6d382e171b003416f51a/board) in ZenHub.
 
+### Creating bugs
+
+Bugs can be created in github
+[here](https://github.com/fleetdm/fleet/issues/new?assignees=&labels=bug%2C%3Areproduce&projects=&template=bug-report.md&title=).
+You should fill out the bug template and place the correct labels on the bug. If this bug is
+reported by a customer you should add the customer codename label, which can be found [here](https://github.com/fleetdm/confidential/labels?q=customer-)
```

Comment:
- GitHub
- skip the newlines
- reported by a customer, add the appropriate customer codename label, which can be found here
- link to https://fleetdm.com/handbook/customers#customer-codenames instead (although that section of the handbook needs a link to what you've done here)

> context:  i'm trying to get faster at these reviews

<img width="638" alt="image" src="https://github.com/fleetdm/fleet/assets/618009/88c81e8a-2c45-4b69-9339-b74aacd6653e">